.github/dependabot.yml

```diff
@@ -401,3 +401,26 @@ updates:
       - "dependencies"
       - "pip"
       - "terraform"
+  # Tezosqa
+  - package-ecosystem: "docker"
```

Comment:
We don't need to keep 2 versions, remove the 0.1 one and keep the 1.0 one.

Reply:
ah then the script and the subdirectories are useless and you can go back to a single root Dockerfile if you want @fleandrei , sorry. Let's say you now know how to it. 😄